lib/src/node/options.dart

```diff
@@ -7,4 +7,6 @@ import 'package:js/js.dart';
 @JS()
 class NodeOptions {
   external String get file;
+  external String get indentType;
+  external int get indentWidth;
```

Comment:
It looks like node-sass supports String indent widths as well, and just parses them into ints. That seems ridiculous to me, but we should probably match their behavior.